packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -196,12 +196,36 @@ export class Relayer {
         });
       else throw receipt;
     } catch (error) {
-      this.logger.error({ at: "InsuredBridgeRelayer#Relayer", type: "Something went wrong slow relaying!", error });
+      this.logger.error({ at: "InsuredBridgeRelayer#Relayer", type: "Something errored slow relaying!", error });
     }
   }
 
-  private async speedUpRelay(/* deposit: Deposit*/) {
-    // TODO: implement
+  private async speedUpRelay(deposit: Deposit) {
+    const bridgePool = this.l1Client.getBridgePoolForDeposit(deposit);
+    const SpeedUpRelayTx = bridgePool.methods.speedUpRelay(deposit as any);
+
+    await this.gasEstimator.update();
+    try {
+      const { receipt, transactionConfig } = await runTransaction({
+        web3: this.l1Client.l1Web3,
+        transaction: (SpeedUpRelayTx as unknown) as TransactionType,
+        transactionConfig: { gasPrice: this.gasEstimator.getCurrentFastPrice().toString(), from: this.account },
+        availableAccounts: 1,
+      });
+
+      if (receipt.events)
+        this.logger.info({
+          at: "InsuredBridgeRelayer#Relayer",
+          type: "Relay instantly sped up 🚀",
```

Comment:
nit: isn't it redundant to say the relay was instantly sped up? Why not just say "Relay sped up"